galaxy_ng/app/migrations/0034_remove_inbound_repos.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+
+def remove_inbound_repos(apps, schema_editor):
+    """Remove inbound repositories and point distribution to staging repository"""
+
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    AnsibleRepository = apps.get_model('ansible', 'AnsibleRepository')
+    RepositoryContent = apps.get_model('core', 'RepositoryContent')
+
+    repos = AnsibleRepository.objects.filter(name__startswith="inbound-")
+
+    staging_repo = AnsibleRepository.objects.get(name="staging")
+
+    AnsibleDistribution.objects.filter(name__startswith="inbound-").update(
```

Comment:
We don't want to delete the inbound distributions because this will break backwards compatibility. They should point to the staging repository instead.